pkg/bitcoin/electrum/electrum_integration_test.go

```diff
@@ -345,6 +345,24 @@ func TestGetTransactionsForPublicKeyHash_Integration(t *testing.T) {
 	}
 }
 
+func TestEstimateSatPerVByteFee_Integration(t *testing.T) {
```

Comment:
This test has a similar problem as [this one](https://github.com/keep-network/keep-core/pull/3578#discussion_r1200310615). I do not consider it blocking this PR but we should investigate what is happening and it it is not a quick fix, at least leave a TODO. 

I altered the host name in `fulcrum ssl` test to something that can't be resolved and while all other tests fail nicely with `electrum_integration_test.go:369: failed to initialize electrum client: [retry timeout [1m0s] exceeded; most recent error: [dial tcp: lookup blackie.c3-softfff.com: no such host]]` , this one gets stuck:

```
=== RUN   TestEstimateSatPerVByteFee_Integration/fulcrum_ssl
panic: test timed out after 10m0s

goroutine 429 [running]:
testing.(*M).startAlarm.func1()
        /opt/homebrew/Cellar/go@1.18/1.18.5/libexec/src/testing/testing.go:2029 +0x8c
created by time.goFunc
        /opt/homebrew/Cellar/go@1.18/1.18.5/libexec/src/time/sleep.go:176 +0x3c

goroutine 1 [chan receive, 2 minutes]:
testing.(*T).Run(0x14000555520, {0x102e1e49b?, 0x1c114841bbb5b?}, 0x103160588)
```

It is not a problem as long as `EstimateSatPerVByteFee` is not used by an automated tool but we should be aware of the limitation and find what is the reason.

Reply:
See 9044c33e